src/SampleServerClasses/MarshallableClass.cs

```diff
@@ -104,7 +98,16 @@ public virtual void DoCallback()
 
 		public virtual void DoCallbackOnEvent(string msg)
 		{
-			AnEventInternal?.Invoke(msg, Name);
+			AnEvent4?.Invoke(msg, msg, msg, Name);
+			AnEvent3?.Invoke(msg, msg, Name);
+			AnEvent2?.Invoke(msg, Name);
+			AnEvent1?.Invoke(msg);
+			AnEvent0?.Invoke();
+		}
+
+		public void DoCallbackOnEvent5(string msg)
+		{
+			AnEvent5?.Invoke(string.Empty, string.Empty, string.Empty, string.Empty, string.Empty);
```

Comment:
Is this intended? The parameter isn't used, which may result in the test being useless.